completions/lintian

```diff
@@ -95,10 +95,10 @@ _comp_cmd_lintian()
                 ;;
             --tags-from-file | --suppress-tags-from-file | --cfg | -p | \
                 --packages-file)
-                _filedir
+                _comp_compgen filedir
```

Comment:
Agree on the suggestion in https://github.com/scop/bash-completion/pull/952#issuecomment-1535594464

Reply:
Fixed in the (forced) push [`bbb0e5f...253609a`](https://github.com/scop/bash-completion/compare/bbb0e5f16a535667a22279ce2fe6e87b2d080c2b..253609ad04003fafcca5e9fcbf44fc8af70ae44a)